app/main/templates/index.html

```diff
@@ -3,117 +3,86 @@
+                    <li style="padding-inline-start: 2.5rem" class="p-front-page lead">Slice position</li>
+                    <li style="padding-inline-start: 2.5rem" class="p-front-page lead">Slice width</li>
+                    <li style="padding-inline-start: 2.5rem" class="p-front-page lead">Uniformity</li>
+                    <li style="padding-inline-start: 2.5rem" class="p-front-page lead">Ghosting</li>
+                </p>
             </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'ghost.jpg') }}" class="card-img-top">
-          <div class="card-body">
-            <h5 class="card-title">Ghosting</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://hazen.readthedocs.io/en/latest/tasks.html#ghosting" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'sp_res.jpg') }}" class="card-img-top">
-          <div class="card-body">
-            <h5 class="card-title">Spatial Resolution</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://hazen.readthedocs.io/en/latest/tasks.html#spatial-resolution" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
-            </div>
-          </div>
-        </div>
-      </div>
-      <div class="col-sm-4">
-        <div class="card" style="width: 18rem;">
-          <img src="{{url_for('static', filename = 'geometric_d.png') }}" class="card-img-top">
-          <div class="card-body">
-            <h5 class="card-title">Geometric Distortion</h5>
-            <h6 class="card-subtitle mb-2 text-muted"></h6>
-            <p class="card-text"></p>
-            <div class="card-footer">
-              <a href="https://readthedocs.org/projects/hazen/" class="btn mr-2"><i class="fas fa-link"></i>Docs</a>
-              <a href="https://github.com/GSTT-CSC/hazen-web-app" class="btn"><i class="fab fa-github"></i>GitHub</a>
+
+            <div class="hazen-dotted-horizontal-line py-3"></div>
+        </section>
+
+        <!-- Example Images -->
+        <section class="pb-5">
+            <div class="row py-3">
+                <div class="col">
+                    <img src="{{url_for('static', filename = 'sp_res.jpg') }}" class="card-img-top">
```

Comment:
I think it would be helpful to add a caption for these example images about which phantom and task they came from

Reply:
A written heading for this section as well

Reply:
Amended.